apps/codelab/src/app/codelabs/extra/feedback-page/feedback-page.module.ts

```diff
@@ -13,6 +13,14 @@ import { CommonModule } from '@angular/common';
 import { SlidesModule } from '@codelab/slides';
 import { environment } from '../../../../environments/environment';
 import { DateRangeComponent } from './date-range/date-range.component';
+import { FeedbackMessageTableComponent } from './feedback-message-table/feedback-message-table.component';
+
+import {
+  MatButtonModule, MatCardModule, MatMenuModule, MatSelectModule,
+  MatTableModule, MatIconModule, MatDatepickerModule, MatNativeDateModule,
+  MatInputModule, MatFormFieldModule, MatExpansionModule
+} from '@angular/material';
```

Comment:
I think those modules should be imported from different paths, e.g.

`import { MatSelectModule } from '@angular/material/select';`